tests/test_oper/test_size_2d.f

```diff
@@ -0,0 +1,49 @@
+
+c
+c     Include file to dimension static arrays
+c     and to set some hardwired run-time parameters
+c
+c     test_oper
+c     Autogenerated using snek5000.operators.Operators.write_size()
+c
+c
+      integer ldim,lx1,lxd,lx2,lx1m,lelg,lelt,lpmin,lpmax,ldimt
+      integer lpelt,lbelt,toteq,lcvelt
+      integer lelx,lely,lelz,mxprev,lgmres,lorder,lhis
+      integer maxobj,lpert,nsessmax,lxo
+      integer lfdm,ldimt_proj
+
+      ! BASIC
+      parameter (ldim=2)                ! domain dimension (2 or 3)
+      parameter (lx1=15)                ! GLL points per element along each direction
+      parameter (lxd=24)                ! GL  points for over-integration (dealiasing)
+      parameter (lx2=15)                ! GLL points for pressure (lx1 or lx1-2)
+
+      parameter (lelg=256)              ! max total number of elements
+      parameter (lpmin=4)               ! min MPI ranks
+      parameter (lpmax=32)              ! max MPI ranks
+      parameter (ldimt=1)               ! max auxiliary fields (temperature + scalars)
+
+      ! OPTIONAL
+      parameter (ldimt_proj=1)          ! max auxiliary fields residual projection
+      parameter (lhis=100)              ! max history/monitoring points
+      parameter (maxobj=1)              ! max number of objects
+      parameter (lpert=1)               ! max number of perturbations
+      parameter (toteq=1)               ! max number of conserved scalars in CMT
+      parameter (nsessmax=1)            ! max sessions to NEKNEK
+      parameter (lxo=15)                ! max GLL points on output (lxo>=lx1)
+      parameter (mxprev=20)
+      parameter (lgmres=30)             ! max dim of projection & Krylov space
+      parameter (lorder=3)              ! max order in time
+      parameter (lx1m=1)                ! GLL points mesh solver
+      parameter (lfdm=0)                ! set to 1 for fast diagonalization method
+      parameter (lelx=16)               ! global tensor mesh dimensions
+      parameter (lely=16)
+      parameter (lelt=64)               ! max number of local elements per MPI rank
+      parameter (lbelt=1)               ! set to lelt for mhd
+      parameter (lpelt=1)               ! set to lelt for linear stability
+      parameter (lcvelt=1)              ! set to lelt for cvode
+
+      ! INTERNALS
+      include 'SIZE.inc'
```

Comment:
Copied from https://github.com/paugier/nek5000-convection-box/blob/9cc3a6436542be4ba0453be6b985a75ea013db01/SIZE with minor changes
- lelr is missing here
- lelx, lely are different